src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparing(String::length, Comparator.reverseOrder()))
```

Comment:
```suggestion
                .sorted(Comparator.comparing(String::length).reversed())
```

추가로 중복되는 단어가 많으므로 distinct() 를 filter() 전에 호출해볼수도 있을것 같습니다.